src/main/java/netzbegruenung/keycloak/authenticator/gateway/ApiSmsService.java

```diff
@@ -140,7 +140,7 @@ private static String clean_phone_number(String phone_number, String countrycode
 		if (countrycode == "") {
 			return phone_number;
 		}
-		String country_number = countrycode.replaceFirst("+", "");
+		String country_number = countrycode.replaceFirst("\\+", "");
```

Comment:
Thanks for catching this! New languages can be surprising ;-)

Reply:
At the first glance I saw no regex too ;)